addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java

```diff
@@ -111,8 +111,8 @@ public WifiSocketPacketConverter() {
             bEncrypted = this.silvercrestEncryptCipher.doFinal(inputByte);
             int encryptDataLength = bEncrypted.length;
 
-            logger.trace("Encrypted data={" + byteArrayToHexString(inputByte) + "}");
-            logger.trace("Decrypted data={" + byteArrayToHexString(bEncrypted) + "}");
+            logger.trace("Encrypted data={{}}", byteArrayToHexString(inputByte));
+            logger.trace("Decrypted data={{}}", byteArrayToHexString(bEncrypted));
```

Comment:
You might run a trouble here with the existing { I don't know if this just works, would require some testing or just replace with square brackets